docs/source/raft_ann_benchmarks.md

```diff
@@ -287,7 +291,7 @@ Configuration files already exist for the following list of the million-scale da
 | `glove-100-angular` | 1.1M | 100 | 10K | Angular |
 | `mnist-784-euclidean` | 60K | 784 | 10K | Euclidean |
 | `nytimes-256-angular` | 290K | 256 | 10K | Angular |
-| `sift-128-euclidean` | 1M | 128 | 10K | Euclidean|
+| `sift-128-euclidean` | 1M | 128 | 10K | Euclidean|**
```

Comment:
This change here should also be reverted

Reply:
lol, yep it certainly should